core/src/cluster_info.rs

```diff
@@ -346,7 +346,8 @@ pub fn make_accounts_hashes_message(
 }
 
 // TODO These messages should go through the gpu pipeline for spam filtering
-#[derive(Serialize, Deserialize, Debug)]
+#[frozen_abi(digest = "6qRS1ZwydpdSqzeyRdDvn5uwfDdFYkuUz4K4jSkd1oFW")]
```

Comment:
FYI: @carllin When you change any `struct`s and `enum`s indirectly referenced by `Protocol`. This digest value needs to be updated after checking the abi change details by: https://github.com/solana-labs/solana/pull/10335/files#r433615921